website/docs/plugin/framework/resources/delete.mdx

```diff
@@ -6,24 +6,24 @@ description: >-
 
 # Delete Resources
 
-Deletion is part of the basic Terraform lifecycle for managing resources. During the [`terraform apply` command](/cli/commands/apply), Terraform calls the provider [`ApplyResourceChange`]((/plugin/framework/internals/rpcs#applyresourcechange-rpc)) RPC, in which the framework calls the [`resource.Resource` interface `Delete` method](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/resource#Resource.Delete). The request contains Terraform prior state data. The response is only for returning diagnostics. The data is defined by the [schema](/plugin/framework/schemas) of the resource.
+Deletion is part of the basic Terraform lifecycle for managing resources. During the [`terraform apply` command](/terraform/cli/commands/apply), Terraform calls the provider [`ApplyResourceChange`](https://terraform.io/(/plugin/framework/internals/rpcs#applyresourcechange-rpc)) RPC, in which the framework calls the [`resource.Resource` interface `Delete` method](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/resource#Resource.Delete). The request contains Terraform prior state data. The response is only for returning diagnostics. The data is defined by the [schema](/terraform/plugin/framework/schemas) of the resource.
```

Comment:
🔴 The `ApplyResourceChange` link syntax is incorrect (extra set of parentheses), causing the link to be prefixed with the .io hostname